components/admin_console/set_by_env.jsx

```diff
@@ -4,7 +4,7 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-export default function() {
+export default () => {
```

Comment:
This could also use a proper display name

Reply:
done